department-of-veterans-affairs/caseflow
# Ensuring save completes before loading the page

Connects #3968 

To test:
1) Add a settimeout in the API call to save the worksheet
2) Edit something on the worksheet
3) Click 'Save to PDF'
4) Ensure that the changes are shown on the PDF
